examples/mlserver/models/text-classification-model/models.py

```diff
@@ -0,0 +1,19 @@
+from mlserver import MLModel
+from mlserver.codecs import decode_args
+from typing import List
+from deepsparse import Pipeline
+
+class DeepSparseRuntime(MLModel):
+    async def load(self) -> bool:
+        self._pipeline = Pipeline.create(
+            task = self._settings.parameters.task,
+            model_path = self._settings.parameters.model_path,
+            batch_size = self._settings.parameters.batch_size,
+            sequence_length = self._settings.parameters.sequence_length,
```

Comment:
is there a place for generic kwargs in the settings? Would be cool if we could use that instead to dump extra pipeline args so we can get full generic pipeline support out of the box